R/indentify.r

```diff
@@ -120,14 +111,12 @@ setMethod("detect_flightline", "LAS",
 #' @return Return nothing. The original object is modified in place by reference.
 #' @export detect_scanline
 #' @importFrom data.table setorder
-setGeneric("detect_scanline", function(obj){standardGeneric("detect_scanline")})
+setGeneric("detect_scanline", function(obj, dt = 30){standardGeneric("detect_scanline")})
 
 #' @rdname detect_scanline
 setMethod("detect_scanline", "LAS",
   function(obj)
   {
-    gpstime <- scanlineID <- ScanDirectionFlag <- NULL
```

Comment:
Make sense, gpstime the moment the laser was shot and not the moment its reflection(s) were registered.

You probably used an old version of the code. Code often changes. Update your fork before changing something is a good practice.

Thanks for this good update.